src/cpp/session/modules/SessionHelp.R

```diff
@@ -760,3 +760,11 @@ options(help_type = "html")
    fmt <- "help/doc/html/Search?pattern=%s&title=1&keyword=1&alias=1"
    sprintf(fmt, utils::URLencode(query, reserved = TRUE))
 })
+
+.rs.addFunction("Rd2HTML", function(file, package = "", Rdmacros = "")
+{
+   tf <- tempfile(); on.exit(unlink(tf))
+   macros <- suppressWarnings(tools:::initialRdMacros(Rdmacros))
+   tools::Rd2HTML(file, out = tf, package = package, macros = macros)
```

Comment:
This mimics what eventually gets called by `R CMD Rdconv` : 

https://github.com/r-devel/r-svn/blob/e90a8c9ccf70236ec5ca74d0d0a055c2952ab9e2/src/library/tools/R/Rd2pdf.R#L530-L533